smEntities/src/main/kotlin/me/ixhbinphoenix/smPl/smEntities/entities/hostiles/Zombie.kt

```diff
@@ -0,0 +1,53 @@
+package me.ixhbinphoenix.smPl.smEntities.entities.hostiles
+
+import me.ixhbinphoenix.smPl.smCore.commands.BaseCommand
+import me.ixhbinphoenix.smPl.smEntities.entities.updateName
+import org.bukkit.Location
+import org.bukkit.World
+import org.bukkit.command.Command
+import org.bukkit.command.CommandExecutor
+import org.bukkit.command.CommandSender
+import org.bukkit.entity.Damageable
+import org.bukkit.entity.Entity
+import org.bukkit.entity.EntityType
+import org.bukkit.entity.Player
+
+class Zombie : Hostiles(), CommandExecutor, BaseCommand {
```

Comment:
BaseCommand is just an interface extending CommandExecutor and TabCompletor, so the extra Implementation of CommandExecutor is not needed